pkgs/tools/inputmethods/fcitx5/fcitx5-anthy.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, stdenv, fetchurl, cmake, extra-cmake-modules, pkg-config, fcitx5, anthy, gettext }:
+
+stdenv.mkDerivation rec {
+  pname = "fcitx5-anthy";
+  version = "5.0.13";
+
+  src = fetchurl {
+    url = "https://download.fcitx-im.org/fcitx5/fcitx5-anthy/${pname}-${version}.tar.xz";
+    sha256 = "EGpyYiNalm4En3Oa++hyOSPLhjbkVcDyZt3LqamfSOI=";
```

Comment:
This looks like an SRI hash but is missing the `sha256-` prefix. Is this hash right?

Reply:
Yes, this hash is right, I don't think `fetchurl` cares whether you have the `sha256-` prefix or not? It built just fine on my system when using this derivation in my config. I'll add the prefix.

Reply:
Ah you specified `sha256 =` instead of `hash =`. I should have looked more closely :)

You can put bare, base64 hashes in `sha256` but it is best to leave the prefix so someone else who comes along isn't momentarily puzzled too